core/src/main/java/io/grpc/LoadBalancer.java

```diff
@@ -75,8 +75,7 @@ public void shutdown() { }
    * @param servers the resolved server addresses, never empty.
```

Comment:
The first paragraph of the javadoc needs to be updated to mention `ResolvedServerInfoGroup`, by saying that addresses in each "group" should be considered equivalent, e.g., pointing to the same server.